distribution/ddf-branding-plugin/src/main/java/org/codice/ddf/branding/impl/DdfBrandingPlugin.java

```diff
@@ -0,0 +1,147 @@
+/**
+ * Copyright (c) Codice Foundation
+ * <p>
+ * This is free software: you can redistribute it and/or modify it under the terms of the GNU Lesser
+ * General Public License as published by the Free Software Foundation, either version 3 of the
+ * License, or any later version.
+ * <p>
+ * This program is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without
+ * even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details. A copy of the GNU Lesser General Public License
+ * is distributed along with this program and can be found at
+ * <http://www.gnu.org/licenses/lgpl.html>.
+ **/
+
+package org.codice.ddf.branding.impl;
+
+import java.io.IOException;
+import java.util.Base64;
+
+import org.apache.commons.configuration.ConfigurationException;
+import org.apache.commons.configuration.PropertiesConfiguration;
+import org.apache.commons.io.IOUtils;
+import org.codice.ddf.branding.BrandingPlugin;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+public class DdfBrandingPlugin implements BrandingPlugin {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(DdfBrandingPlugin.class);
+
+    private static String brandingProperties = "/META-INF/branding.properties";
```

Comment:
I would suggest making this non-static, replacing its setter with a constructor that takes the value, and rename this field to something that make it clearer that this is the branding property file path.